db/schema.rb

```diff
@@ -11,17 +11,28 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160503184017) do
+ActiveRecord::Schema.define(version: 20160503231211) do
+
+  create_table "features", force: :cascade do |t|
+    t.string   "name"
+    t.integer  "user_id",    null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  add_index "features", ["user_id"], name: "index_features_on_user_id"
 
   create_table "metrics", force: :cascade do |t|
     t.string   "name"
     t.string   "definition"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.integer  "user_id"
+    t.integer  "user_id",    null: false
```

Comment:
Deleting the migration file `db/migrate/20160503184017_add_user_to_metrics.rb` doesn't drop the column it had already added to the development db. 

`schema.rb` is generated from the current structure of the development db.

As a result, `schema.rb` still contains the `metrics.user_id` column (line 30).

To avoid this in future, favor writing new migrations to undo unwanted, old migrations. See `remove_reference`/`remove_column` in the Rails API docs.